ec2uploadimg

```diff
@@ -645,6 +659,9 @@ try:
                 elif args.rootSwapMethod:
                     ami = uploader.create_image_use_root_swap(args.source)
                     print('Created image: ', ami)
+                elif args.useSnap:
+                    ami = uploader.create_image_from_snapshot(args.source)
+                    print('Created image from snapshot: ', ami)
```

Comment:
Ultimately all images are created from a snapshot thus the message when the code exists successfully should remain the same as "Created image: "

Reply:
Thanks I fixed both of these and did a new push